include/deal.II/distributed/tria.h

```diff
@@ -169,19 +169,19 @@ namespace parallel
      * {
      *   ... create the coarse mesh ...
      *
-     *   coarse_grid.signals.post_refinement.connect(
-     *     std::bind (&set_boundary_ids<dim>, std::ref(coarse_grid)));
+     *   coarse_grid.signals.post_refinement.connect([&coarse_grid](){
+     *     set_boundary_ids<dim>(coarse_grid);});
```

Comment:
Can you just wrap the `[&coarse_grid](){` over to the beginning of the next line? I know that our indentation tool does it that way, but it's just unreadable.